task_DEV_6/task_DEV_6/Stack.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+
+namespace task_DEV_6
+{
+    class Stack
```

Comment:
Bad name. We need to have possibility to distinguish original `class Stack` from our `Stack`.

Reply:
Done.